### PR TITLE
Update readme with Scala import settings

### PR DESCRIPTION
**Pull request by roberto-orlandi-sonarsource**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -202,6 +202,14 @@ Once installed set the files as such:
...
 ![Intellij javascript onsave](intellij/intellij-javscript-actionsonsave.png)
 
+For Scala:
+
+Go to `Preferences/Settings > Editor > Code Style > Scala > Imports` and update:
+* Class count to use import with '_' -> 999
+* Ensure `Merge imports with the same prefix into one statement` is checked
+
+Then go to `Preferences/Settings > Editor > General > Auto import` and uncheck "Optimize imports on the fly" for Scala. This is necessary because there are some imports that are not explicitly used in the code but are necessary for the code to compile.
+
```
Comment (roberto-orlandi-sonarsource) [inline README.md:212]: I just added the settings that I know are relevant to what we want to brain. There are other things that are checked and I could not see a difference in the behavior of the autoimport, so I did not mention them here, I guess we can update this if we see in the future new required settings

File: README.md [hunk 2]
```diff
@@ -202,6 +202,14 @@ Once installed set the files as such:
 Then to make sure prettier is run correctly, modify the on save actions:
 ![Intellij javascript onsave](intellij/intellij-javscript-actionsonsave.png)
 
+For Scala:
+
+Go to `Preferences/Settings > Editor > Code Style > Scala > Imports` and update:
+* Class count to use import with '_' -> 999
+* Ensure `Merge imports with the same prefix into one statement` is checked
+
+Then go to `Preferences/Settings > Editor > General > Auto import` and uncheck "Optimize imports on the fly" for Scala. This is necessary because there are some imports that are not explicitly used in the code but are necessary for the code to compile.
```
Comment (quentin-jaquier-sonarsource) [inline README.md:211]: I'm wondering if this should be in the common configuration.
I think the behavior we face in Armor (import that looks unused while required for the compilation) is a bit specific, it should not impact any Scala project.
What do you think about moving this sentence to the Armor Readme instead?

Comment (roberto-orlandi-sonarsource) [reply README.md:211]: Fair point. I was not aware of this and assumed that was a common Scala scenario. I agree with you then. I will remove it from here, and you can maybe add it in the readme while you are on your ticket. Or if you think it does not belong there, I can add it (now or once you are done with your ticket)

Comment (quentin-jaquier-sonarsource) [reply README.md]: I will add it in the context of my ticket.

**Review by quentin-jaquier-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**